### PR TITLE
Added missing textures_image_kernel example in web makefile

### PR DESCRIPTION
**Pull request by Zapunidi**
The PLATFORM_WEB compilation failed on textures_image_kernel. I figured out that it was missing in Makefile, so I just made similar lines in Makefile.Web. There may be a need to limit RAM usage. I have no expertise to decide that.

### CODE REVIEW

**Comment by raysan5** [comment]
@Zapunidi Thanks for the review! That example was just added yesterday and not properly reviewed yet, the web compilation also requires adding the required resource image.

File: examples/Makefile.Web [hunk 1]
```diff
@@ -702,6 +703,9 @@ textures/textures_image_drawing: textures/textures_image_drawing.c
 
 textures/textures_image_generation: textures/textures_image_generation.c
 	$(CC) -o $@$(EXT) $< $(CFLAGS) $(INCLUDE_PATHS) $(LDFLAGS) $(LDLIBS) -D$(PLATFORM) -s TOTAL_MEMORY=67108864
+	
+textures/textures_image_kernel: textures/textures_image_kernel.c
+	$(CC) -o $@$(EXT) $< $(CFLAGS) $(INCLUDE_PATHS) $(LDFLAGS) $(LDLIBS) -D$(PLATFORM)
```
Comment (raysan5) [inline examples/Makefile.Web:708]: Required image resource used by example, I think it's cat.png, needs to be added to compilation process

**Comment by raysan5** [comment]
@Zapunidi thanks for the review!

**Comment by Zapunidi** [comment]
I added image resource. I checked that it is indeed cat.png. I checked that web compiled example is working on my webserver.

> @Zapunidi thanks for the review!

Glad to help. I liked your raylib.

**Pull request merged**